content/blog/por-onde-comecar.md

```diff
@@ -28,21 +28,21 @@ Comece criando um programinha conhecido tradicionalmente como "Olá Mundo". Impr
 print("Olá Mundo.")
 ```
 
-Uma vez entendido o funcionamento do programa "Olá Mundo" você criar um programa com duas instruções conforme exemplo abaixo:
+Uma vez entendido o funcionamento do programa "Olá Mundo" você poderá criar um programa com duas instruções, conforme o exemplo abaixo:
 
 ```python
 print("Olá Mundo.")
 print("Escreva algo aqui.")
 ```
 
-Uma vez entendido o conceito de sequência de instruções, o próximo passo é usar variáveis. Crie um programa, conforme o exemplo abaixo. e atribua o valor 0 (zero) a uma variável chamada a e exiba o valor de a na tela do seu computador.
+Uma vez entendido o conceito de sequência de instruções, o próximo passo é usar variáveis. Crie um programa conforme o exemplo abaixo e atribua o valor 0 (zero) a uma variável chamada _a_ e exiba o valor de _a_ na tela do seu computador.
```

Comment:
já que voe está colocando a variavel em itálicos, talvez seja uma boa oportunidade pra colocar em `preformatted`.